crates/bevy_pbr/src/light/directional_light.rs

```diff
@@ -71,6 +110,7 @@ impl Default for DirectionalLight {
             color: Color::WHITE,
             illuminance: light_consts::lux::AMBIENT_DAYLIGHT,
             shadows_enabled: false,
+            soft_shadow_size: None,
```

Comment:
Could we add a constant to make it easy to set the Sun's diameter?

Reply:
I believe that using the sun's diameter here would not be particularly useful unless the sun were its actual physical distance away from the target, which would be impractical.